members/C001047.yaml

```diff
@@ -3,6 +3,10 @@ contact_form:
   method: post
   steps:
     - visit: "https://www.capito.senate.gov/contact/share-your-opinion"
+    - find:
```

Comment:
Does the alert appear on submission? Or is it intermittent on load? I tried accessing that URL and aren't seeing the alert so I wonder if this will fail if it can't find that alert modal.

Reply:
If the alert doesn't show up it will cause the find step to wait for the full duration (5 seconds). Then the click on step after will fail. because of the funky way yamltron parses these YAMLs having the click fail will not prevent the rest of the yaml's steps from being attempted. This ultimately just causing it to take longer, but should still work in the end

Reply:
@vramoscoreas it seems to appear both on load and submission. Tried loading the page in incognito (if you haven't yet). Even though it also appears on submission, yamltron is able to find the success message without having to exit out of the modal. Thus why I didn't add anything extra at the end of this yaml. 

RE the initial `find` failing, what Tommy said